### PR TITLE
update speechio whisper ft results

### PR DESCRIPTION
**Pull request by yuekaizhang**
Fine-tuning whisper-large-v2 using multi-hans-zh dataset, exclude datatang 200h (which is not open sourced any more), updating wenetspeech (according to https://github.com/wenet-e2e/WenetSpeech/discussions/54)

Currently, rank 8 according to https://github.com/SpeechColab/Leaderboard

| Rank 排名 | Model 模型 | CER 字错误率 | Date 时间 |
| --- | --- | --- | --- |
| 1 | ximalaya_api_zh | 1.72% | 2023.12 |
| 2 | aliyun_ftasr_api_zh | 1.85% | 2023.12 |
| 3 | microsoft_batch_zh | 2.40% | 2023.12 |
| 4 | bilibili_api_zh | 2.90% | 2023.09 |
| 5 | tencent_api_zh | 3.18% | 2023.12 |
| 6 | iflytek_lfasr_api_zh | 3.32% | 2023.12 |
| 7 | aispeech_api_zh | 3.62% | 2023.12 |
| 8 | **whisper-large-ft-v1** | **4.45%** | 2024.04 |
| 9 | baidu_pro_api_zh | 7.29% | 2023.12 |

|                | Split         | Greedy Search |
|----------------|---------------|---------------|
| Datasets       |               |               |
| alimeeting     | eval          | 23.45         |
| alimeeting     | test          | 25.42         |
| aishell-1      | dev           | 0.78          |
| aishell-1      | test          | 0.83          |
| aishell-2      | dev           | 2.75          |
| aishell-2      | test          | 2.93          |
| aishell-4      | test          | 17.11         |
| magicdata      | dev           | 2.68          |
| magicdata      | test          | 2.33          |
| kespeech-asr   | dev phase1    | 4.97          |
| kespeech-asr   | dev phase2    | 2.02          |
| kespeech-asr   | test          | 6.34          |
| WenetSpeech    | dev           | 5.06          |
| WenetSpeech    | test meeting  | 8.38          |
| WenetSpeech    | test net      | 6.94          |

### CODE REVIEW

**Comment by JinZr** [comment]
thank you so much!

i'll look into this pr tonight!

File: egs/multi_zh-hans/ASR/whisper/train.py [hunk 1]
```diff
@@ -146,7 +147,7 @@ def get_parser():
         "--model-name",
         type=str,
         default="large-v2",
-        choices=["large-v2", "large-v3", "medium", "small", "base", "tiny"],
```
Comment (JinZr) [inline egs/multi_zh-hans/ASR/whisper/train.py:149]: should we keep the "base" option here and also in the ``decode.py``?

File: egs/multi_zh-hans/ASR/prepare.sh [hunk 2]
```diff
@@ -299,15 +299,6 @@ if [ $stage -le 12 ] && [ $stop_stage -ge 12 ]; then
     log "Compute KeSpeech fbank for test/dev"
     ./local/compute_fbank_kespeech_dev_test.py
 
-    if [ ! -f data/fbank/kespeech/kespeech-asr_cuts_train_phase1.jsonl.gz ]; then
```
Comment (JinZr) [inline egs/multi_zh-hans/ASR/prepare.sh:302]: we need line 302 to 310 to combine the cut set pieces of kespeech

File: egs/multi_zh-hans/ASR/whisper/train.py [hunk 3]
```diff
@@ -824,7 +870,7 @@ def run(rank, world_size, args):
 
     if params.print_diagnostics:
         opts = diagnostics.TensorDiagnosticOptions(
-            512
```
Comment (JinZr) [inline egs/multi_zh-hans/ASR/whisper/train.py:827]: the number here should be 512 to align with PR #1269

File: egs/wenetspeech/ASR/whisper/train.py [hunk 4]
```diff
@@ -803,7 +806,7 @@ def run(rank, world_size, args):
 
     if params.print_diagnostics:
         opts = diagnostics.TensorDiagnosticOptions(
-            512
```
Comment (JinZr) [inline egs/wenetspeech/ASR/whisper/train.py:806]: this number should also be 512 to align with PR #1269

File: egs/wenetspeech/ASR/whisper/train.py [hunk 5]
```diff
@@ -145,7 +146,7 @@ def get_parser():
         "--model-name",
         type=str,
         default="large-v2",
-        choices=["large-v2", "large-v3", "medium", "small", "base", "tiny"],
```
Comment (JinZr) [inline egs/wenetspeech/ASR/whisper/train.py:148]: should we also keep the "base" option here?

**Comment by JinZr** [comment]
Thank you yuekai!

I left a few comments at the PR, please check those at your convenience. 

best
jin

**Comment by yuekaizhang** [comment]
> Thank you yuekai!
> 
> I left a few comments at the PR, please check those at your convenience.
> 
> best jin

@JinZr  Done. Thanks.

**Review by JinZr — APPROVED** [review]
LGTM!

**Pull request merged**